### ISSUE TITLE
[FEATURE] - Support autocomplete input styling

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4216]
### Portfolio

Customer

### ART and Team

Slack: https://porschedev.slack.com/archives/CT7AVHTTQ/p1771942219514289

### Problem

Autocomplete fields which are styled by the browser look a bit broken:
<img width="829" height="746" alt="Image" src="https://github.com/user-attachments/assets/3e35a6a3-90c4-43d4-84ed-27276ce001a2" />

### Solution ideas and examples

Probably fix it with CSS `:autofill` pseudo class: https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Selectors/:autofill

### Business priority

Medium
